File: modules/mrz-reader/ios/MrzReaderModule.swift
// modules/mrz-reader/ios/MrzReaderModule.swift
import ExpoModulesCore

public class MrzReaderModule: Module {
  public func definition() -> ModuleDefinition {
    Name("MrzReader")

    // Defines the native view component.
    View(MrzReaderView.self) {
      // Defines events that the view can send to JavaScript.
      Events("onMrzExtracted", "onError")
    }
  }
}

File: modules/mrz-reader/android/src/main/java/expo/modules/mrzreader/MrzReaderModule.kt
package expo.modules.mrzreader

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class MrzReaderModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("MrzReader")
    View(MrzReaderView::class) {
      Events("onMrzExtracted", "onError")
    }
  }
}
